public/index.js

```diff
@@ -9,6 +9,26 @@ function makeRequest(url, callback) {
   xhr.send();
 }
 
+function render(error, response) {
+  if (error) {
```

Comment:
it would be useful to write a comment to say what type you expect response to be. e.g. an array of strings